docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,71 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+Since tokens are attached to your account and not your machine, you can re-use tokens, even multiple times per day. In this page, we'll discuss how tokens work and how machines are counted against your subscription.
+
+Pro tokens
+----------
+
+- Obtaining tokens:
+    After subscribing to Ubuntu Pro, you can access your tokens through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each token can be used to attach multiple machines depending on the subscription.
+
+- Attaching tokens:
+    Command: Use ``sudo pro attach`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching tokens:
```

Comment:
Detaching in general doesn't involve the token - maybe this section should be moved down to the Machine activity count section

Reply:
We could maybe just leave it here? It helps with the coherence of the topic of pro token related commands and the continuity of the document. This would look a little out of place in the machine activity count section since we don't talk about how detach affects machine counts. What do you think?

Reply:
I agree, I think it makes more sense here